tests/baselines/reference/printerApi/printsNodeCorrectly.emptyGlobalAugmentation.js

```diff
@@ -0,0 +1 @@
+declare global { }
```

Comment:
why does this end up in a .js? I expected a .d.ts or something.

Reply:
the printer does not care what it prints to. the transforms are the one that do.